ImageTransform/CommonPart/unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Gui)
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${GUI_CMAKE_RUNTIME_OUTPUT_DIR_TEST_COMMON})
+
+add_executable(Unit_Test_CommonPart_lib
```

Comment:
It seems you merged static lib with exe idea. Should it be that way?

Reply:
This is correct, this target should be executable, it is a common part unit test.

Reply:
In that case I would like to propose to remove postfix `*_lib`. It brings no additional information.